Cargo.toml

```diff
@@ -22,6 +21,7 @@ exclude = [
     "crates/wasi-component-loader/examples",
     "examples/hooks-template",
     "examples/query_plan",
+    "extensions/rest",
```

Comment:
We can't compile the extension with wasm32-wasip2 target if it's in the workspace. We suddenly start compiling tokio and fail...

Reply:
We might wanna move extensions to their own repo at some point...